resources/views/ManageEvent/Dashboard.blade.php

```diff
@@ -100,8 +100,13 @@
                         <div class="panel-heading panel-default">
                             <h3 class="panel-title">
                                 @lang("Dashboard.ticket_sales_volume")
+<<<<<<< HEAD
```

Comment:
By the looks of things you had a merge conflict and you haven't resolved it.

I think the easiest way to fix this is by a adding a new commit to remove the `<<<<<<< HEAD`, `=======` and `>>>>>>> dcd171a... Correct sales volume in 2 other files` lines.

Reply:
Sorry about that – fixed in 3e13e5a77cee8e6b072fdeb44ef18389eac8681f